### ISSUE TITLE
Activity ideas for the workshop

### ISSUE THREAD

**Issue by maryi** [maryi/CSCW-workshop4#2]
How do you think we should organize the workshop? Any activity ideas?
Please let the team know :)

**Comment by maryi** [comment]
Hi team,
Considering the material we already have on the topic we have two perspectives: 1. the 'good' things you can do with social media and 2. the controversial about using social media data (or its services in general)
I was talking with @areyeslo and we think we can make an interesting combination with that literature.
We could divide our workshop in two sessions.. The first one could talk about the 'goodness of social media' and then the second part we can go and present the ethical challenges it also represents.
Presenting controversial topics at the end will allow us to leave a lot of open questions to the public and discusss with them at the end of the class.
What do you think?

**Comment by areyeslo** [comment]
I agree to have different approaches and finishing with the controversial part (ethics). I am searching more papers about impact of social media on Education. To have impact on Software Engineering, Education and Ethics. What do you think?
For our meeting, I am busy on Friday from 12:00 to 2:00 pm. The remaining day I am free to discuss about the content and coordination of workshop.
